visualize.Rmd

```diff
@@ -1,115 +1,114 @@
-# Data visualisation
+# Visualización de datos
 
-## Introduction
+## Introducción
 
-> "The simple graph has brought more information to the data analyst’s mind 
-> than any other device." --- John Tukey
+>"Un simple gráfico ha brindado más información a la mente del analista de datos
+>que cualquier otro dispositivo". --- John Tukey
 
-This chapter will teach you how to visualise your data using ggplot2. R has several systems for making graphs, but ggplot2 is one of the most elegant and most versatile. ggplot2 implements the __grammar of graphics__, a coherent system for describing and building graphs. With ggplot2, you can do more faster by learning one system and applying it in many places.
+En este capítulo aprenderás cómo visualizar datos usando el paquete **ggplot2**. De los muchos sistemas que posee R para hacer gráficos, **ggplot2** es uno de los más elegantes y versátiles. Esto se debe a que implementa  un sistema coherente para describir y construir gráficos, conocido como la __gramática de gráficos__. Con **ggplot2** puedes hacer más cosas en menor tiempo, aprendiendo un único sistema y aplicándolo en muchos lugares
```

Comment:
Si al comenzar con: En este capítulo aprenderás cómo visualizar [tus] datos usando el paquete **ggplot2**, creo que queda mejor ya que hay un sentido de pertenencia del trabajo que lo rescatan los autores en Inglés (your data instead of just data or some data).

Reply:
En el segundo punto seguido donde dice: " Esto se debe a que implementa  un sistema coherente para describir y construir gráficos"...como proviene de una frase en la que se compara R base y ggplot2, queda un poco suelto quién es el que implementa el sistema coherente.
Yo agregaría:
 Esto se debe a que **ggplot2**  implementa  un sistema coherente para describir y construir gráficos

Reply:
Estoy de acuerdo con los comentarios de @a-saza y @ajcanepa. Faltaría que validación de la parte de @flor14. La versión definitiva del párrafo sería entonces: 

> En este capítulo aprenderás cómo visualizar tus datos usando el paquete **ggplot2**. De los muchos sistemas que posee R para hacer gráficos, **ggplot2** es uno de los más elegantes y versátiles. Esto se debe a que **ggplot2** implementa  un sistema coherente para describir y construir gráficos, conocido como la **gramática de gráficos**. Con **ggplot2** puedes hacer más cosas en menor tiempo, aprendiendo un único sistema y aplicándolo en diferentes ámbitos.